plugins/remark-snackplayer/package.json

```diff
@@ -21,11 +21,10 @@
   "dependencies": {
     "dedent": "^0.7.0",
     "object.fromentries": "^2.0.3",
-    "unist-builder": "^2.0.3",
```

Comment:
we are removing this because we don't need it?

Reply:
This is because it is used for the remark snackplayer code but it only works for MDX v1 and its usage has been removed.

<img width="710" alt="CleanShot 2023-09-12 at 15 01 12@2x" src="https://github.com/facebook/react-native-website/assets/749374/8f40a2e7-0e90-4c53-acf4-be2279474f63">


MDX v1 used to be more based on "string concatenation".

With MDX v2, JSX elements have first-class MDX AST nodes that you can construct manually.

This code has been replaced by: 

<img width="500" alt="CleanShot 2023-09-12 at 15 02 55@2x" src="https://github.com/facebook/react-native-website/assets/749374/65297732-3a28-4a37-9612-d55e71f5b0b2">